core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/configure-postgresql.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Set OS specific paths
+  set_fact:
+    pg_config_dir: "{{ pg_config_dir[ansible_os_family] }}"
+  vars:
+    pg_config_dir:
+      RedHat: /var/opt/rh/rh-postgresql10/lib/pgsql/data
+      Debian: /etc/postgresql/10/main
+
+- name: Initialize database # RedHat specific block
+  block:
+    - name: Check initialization status
+      stat:
+        path: "{{ pg_config_dir }}/pg_hba.conf"
+      register: pg_hba_conf_stat
+
+    - name: Initialize database
+      command: /opt/rh/rh-postgresql10/root/usr/bin/postgresql-setup --initdb
+      when:
+        - not pg_hba_conf_stat.stat.exists
+  when:
+    - ansible_os_family == 'RedHat'
+
+- name: Enable postgresql service
+  systemd:
+    name: postgresql
+    enabled: yes
+
+- name: Change pg_hba.conf
+  replace:
+    path: "{{ pg_config_dir }}/pg_hba.conf"
+    regexp: '^host.*?all.*?all.*?127\.0\.0\.1\/32.*?ident$'
+    replace: 'host    all             all       0.0.0.0/0          md5'
+    backup: yes
+  register: change_pg_hba_conf
+
+- name: Configure pgAudit extension
+  include_tasks: extensions/configure-pgaudit.yml
+  when:
+    - specification.extensions.pgaudit.enabled is defined
```

Comment:
What if pgaudit will be undefined?

Reply:
Then condition will be `false` and task will be skipped.